lib/quadquizaminos_web/live/contests_live.ex

```diff
@@ -140,7 +135,7 @@ defmodule QuadquizaminosWeb.ContestsLive do
 
   def handle_info(:update_component_timer, socket) do
     Enum.each(socket.assigns.contests, fn contest ->
-      if Contests.contest_running?(contest.name) do
+      unless Contests.ended_contest?(contest.id) do
```

Comment:
@TraceyOnim how is this change related to the countdown?

Reply:
At this point, an update is sent to the contest component only if the the contest is active . Due to that, the contest countdown could only count only when the contest is started

Reply:
okay makes sense